bin/sync-failed-browser-test-images

```diff
@@ -0,0 +1,108 @@
+#! /usr/bin/env bash
+
+# DOC: Sync failed github action browser-test snapshots with local system.
+
+function exit_on_jq_failure() {
+  local last_exit_code="${1}"
+  local value="${2}"
+  local message="${3}"
+
+  if [[ "${last_exit_code}" != "0" ]]; then
+    echo "jq command failed. ${message}"
+    exit 1
+  fi
+
+  if [[ "${value}" == "null" ]]; then
+    echo "${message}"
+    exit 1
+  fi
+}
+
+function exit_on_curl_failure() {
+  local last_exit_code="${1}"
+  local url="${2}"
+
+  if [[ "${last_exit_code}" != "0" ]]; then
+    echo "Curl failed trying to call ${url}"
+    exit 1
+  fi
+}
+
+function verify_dependencies() {
+  if ! command -v jq &>/dev/null; then
+    echo "Missing jq command"
+    echo "https://jqlang.github.io/jq"
+    exit 1
+  fi
+
+  if [[ -z "${GITHUB_TOKEN}" ]]; then
+    echo "You need a GitHub access token to download the archive."
+    echo "Set 'GITHUB_TOKEN' environment variable with the value"
+    echo "To create one go here: https://github.com/settings/tokens"
+    exit 1
+  fi
+}
+
+verify_dependencies
+
+# Set path to root of repo
+pushd "$(git rev-parse --show-toplevel)" >/dev/null || exit
```

Comment:
If `git rev-parse` fails there's going to be bigger problems going on. I only added it because I have shellcheck enabled in vim. I added an exit code. If you really want a message I can add it in. https://www.shellcheck.net/wiki/SC2164

Reply:
lgtm